app/src/main/java/com/example/places/ListFragment.kt

```diff
@@ -20,28 +25,42 @@ class ListFragment : Fragment(R.layout.list_fragment) {
         savedInstanceState: Bundle?
     ): View? {
         binding = ListFragmentBinding.inflate(inflater, container, false)
+        binding2 = ListItemBinding.inflate(inflater,container,false)
         return binding?.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        viewModel.placesLiveData.observe(viewLifecycleOwner, { places ->
-            setupAdapter(places)
-        })
+        viewModel.placesLiveData.observe(viewLifecycleOwner) { places ->
+            setupAdapter(places) }
+
+        viewModel.placesLiveData.observe(viewLifecycleOwner) { places ->
```

Comment:
You're observing the same livedata twice. And in this case, you're doing nothing since you're loading the image inside the viewholder. This observer is useless.